models/doctype/Uom/Uom.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
```

Comment:
How do you use this UOM?

UOM does not have an item property. They have a many-to-many relationship, so `Item UOM` should be a separate DocType